src/renderer/App.tsx

```diff
@@ -1,13 +1,14 @@
-import Main from 'electron/main';
 import { MemoryRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
 import LoginPage from './pages/LoginPage';
+import Tabs from './components/Tabs';
 
 export default function App() {
   return (
     <Router>
       <Routes>
         <Route path="/login" element={<LoginPage />} />
+        <Route path="/master/MasterPage" element={<Tabs />} />
```

Comment:
Oh by this I more meant make the path `/master`, then create a new component called `MasterPage` that just returns `<Tabs />` for now, and use that as the element for this route